lib/lotus/helpers/asset_tag_helpers.rb

```diff
@@ -0,0 +1,22 @@
+require 'lotus/helpers'
+
+module Lotus
+  module Helpers
+    # The helper methods used for using assets.
+    #
+    # @since 0.6.0
+    # @api public
+    module AssetTagHelpers
+      include Lotus::Helpers::HtmlHelper
+
+      def image(source, options = {})
+        raise ArgumentError.new("source argument can't be blank") if source.blank?
+
+        options[:src] = "/assets/#{source}"
```

Comment:
@jodosha I believe I would need to use `Lotus::Assets::Configuration.prefix` here, but I am not sure how to access the Assets configuration from here.